examples/printerdemo_mcu/esp-idf/main/idf_component.yml

```diff
@@ -0,0 +1,9 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: MIT
+
+## IDF Component Manager Manifest File
+dependencies:
+  idf: ">=5.0"
+  espressif/esp-box: "^2.3"
```

Comment:
esp-box comes with lvgl. Even though we're not using it, this is a bit of a waste. But this still provide all the functions to initialize the screen and touch.